lib/signs_ui_web/templates/signs/index.html.eex

```diff
@@ -1,8 +1,12 @@
 <h3>Countdown Signs</h3>
 <div>
-  <ul>
+  <%= form_for @conn, signs_path(@conn, :update), [as: :signs], fn f -> %>
     <%= for {sign_id, sign} <- @signs do %>
-    <li><%= sign_id%> : <%= if sign.enabled?, do: "Enabled", else: "Disabled" %></li>
+      <div>
+        <label><%= sign_id %></label>
```

Comment:
You should make it so that clicking on the label will select / deselect the checkbox. I usually do this by just putting the checkbox _within_ the label tag, but you can also go the `for=` route.